project_report/tex/requirements.tex

```diff
@@ -8,6 +8,29 @@ \subsubsection{Functional \& Non-Functional Requirements}
 
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
+The machine is able to produce a different kind of beers depending on the recipe
+used. The beer is produced in batches which have their own batch id. When 
+producing a batch the quantity, speed and batch id can be configured by an 
+external system. When a batch is done the quality inspection system measures 
+the quality for future use. These values are all accessible in the machines 
+SCADA system that monitor and supervises both the machines PLC system and the 
+internal sensors PLC system as well as the quality inspection system.
+
+\myparagraph{Sensors}
+To ensure the most optimal environment the machine is equipped with three 
+sensors that measure while production is happening. This data is useful because 
+it gives insight into the internal environment of the machine while producing 
+products. Currently, it measures temperature, humidity and vibration. On that 
```

Comment:
This needs to be higher up. See other comments.

Reply:
higher up?